examples/cellfinder.py

```diff
@@ -147,45 +102,60 @@ def _info(self):
+                            "offsets": datasets.Sequence([datasets.Value("int32")]),
+                            "normalized": [
+                                {
+                                    "db_name": datasets.Value("string"),
+                                    "db_id": datasets.Value("string"),
+                                }
+                            ],
+                        }
+                    ],
+
+                    "relations": [
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "arg1_id": datasets.Value("string"),
+                            "arg2_id": datasets.Value("string"),
+                            "normalized": [
+                                {
+                                    "db_name": datasets.Value("string"),
+                                    "db_id": datasets.Value("string"),
+                                }
+                            ],
+                        }
+                    ],
+                }
+            )
+
         return datasets.DatasetInfo(
-            # This is the description that will appear on the datasets page.
             description=_DESCRIPTION,
             features=features,
-            # If there's a common (input, target) tuple from the features,
-            # specify them here. They'll be used if as_supervised=True in
-            # builder.as_dataset.
-            supervised_keys=None,
-            # Homepage of the dataset for documentation
             homepage=_HOMEPAGE,
-            # License for the dataset if available
             license=_LICENSE,
-            # Citation for the dataset
             citation=_CITATION,
         )
 
     def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
-        """
-        Step 5: Download and extract the dataset.
-
-        For each config name, run `download_and_extract` from the dl_manager; this will download and unzip any files within a cache directory, specified by `data_dir`.
-
-        `download_and_extract` can accept an iterable object and return the same structure with the url replaced with the path to local files:
-
-        ex:
-        output = dl_manager.download_and_extract({"data1:" "url1", "data2": "url2"})
-
-        output
-        >> {"data1": "path1", "data2": "path2"}
-
-        Nested zip files can be cached also, but make sure to save their path.
-
-        Populate "SplitGenerator" with `name` and `gen_kwargs`. Note:
-
-        - `name` can be: datasets.Split.<TRAIN/TEST/VALIDATION> or a string
-        - all keys in `gen_kwargs` can be passed to `_generate_examples()`. If your dataset has multiple files, you can make a separate key for each file, as shown below:
-
-        """
-
         # No need for a download location; just specify file location
         data_dir = os.getcwd()
```

Comment:
if this is a local dataset, we should get `data_dir` from the `load_dataset` kwarg. 
```suggestion
        data_dir = self.config.data_dir
```